src/styles/blocks/header.scss

```
@@ -0,0 +1,66 @@
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    cursor: pointer;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    justify-content: flex-end;
+    list-style: none;
+    margin: 0;
+    padding-left: 0;
+  }
+
+  &__item {
+    display: inline-block;
+    margin-right: 20px;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link {
+    height: 60px;
+    padding: 0;
+    display: flex;
+    align-items: center;
+    font-weight: 500;
+    position: relative;
+    cursor: pointer;
+    text-decoration: none;
+    font-size: 12px;
+    text-transform: uppercase;
+    color: #{$color-main-accent};
```

why do u can't use something like this?
```suggestion
    color: $color-main-accent;
```